webpack/assets/javascripts/react_app/components/bookmarks/index.js

```diff
@@ -45,12 +41,17 @@ class BookmarkContainer extends React.Component {
         <Dropdown.Toggle />
         <Dropdown.Menu>
           <MenuItem header>{__('Saved Bookmarks')}</MenuItem>
-          {getLoadingStatus(errors, bookmarks) === STATUS.PENDING && (
-            <Loader spinnerSize="xs" status={STATUS.PENDING} />
+          {status === STATUS.PENDING && (
+            <MenuItem className='loader-root'>
+              <Spinner size="xs" loading/>
+            </MenuItem>
           )}
-          {getLoadingStatus(errors, bookmarks) === STATUS.RESOLVED &&
-            bookmarks.map(({ name, query }) => <Bookmark key={name} text={name} query={query} />)}
-          {getLoadingStatus(errors, bookmarks) === STATUS.ERROR &&
+          {status === STATUS.RESOLVED &&
+            ((bookmarks.length > 0 &&
+              bookmarks.map(({ name, query }) => (
+                <Bookmark key={name} text={name} query={query} />
+              ))) || <MenuItem> __('None found')</MenuItem>)}
```

Comment:
This needs to be interpolated, i.e. `<MenuItem>{__('None found')}</MenuItem>`.

![image](https://user-images.githubusercontent.com/4107560/34261668-3637231c-e66a-11e7-87c7-3a72810b3d52.png)

In addition, I think this text should not be clickable. Maybe it's worth marking it as [disabled](https://getbootstrap.com/docs/3.3/components/#dropdowns-disabled).